docs/source-app/api_reference/core.rst

```diff
@@ -0,0 +1,26 @@
+:orphan:
+
+##################
+lightning_app.core
+##################
+
+.. contents::
+    :depth: 1
+    :local:
+    :backlinks: top
+
+.. currentmodule:: lightning_app.core
+
+Core APIs
+___________________
+
+.. autosummary::
+    :toctree: api/
+    :nosignatures:
+    :template: classtemplate.rst
+
+    LightningApp
+    LightningFlow
+    LightningWork
```

Comment:
this is listed in `api_references.rst` too

Reply:
Yes, because it appears in two places. Too the best of my knowledge this can't be shared as reference like some of the other content.